astropy/modeling/fitting.py

```diff
@@ -403,9 +402,9 @@ def objective_function(self, fps, *args):
         _fitter_to_model_params(model, fps)
         meas = args[-1]
         if weights is None:
-            return np.ravel(model(*args[2 : -1]) - meas)
+            return np.ravel(model(*args[2: -1]) - meas)
```

Comment:
Even though existing PEP8 errors are annoying it is generally preferred to leave them alone if they are unrelated to your specific patch.  Then you can do a separate PR that is devoted strictly to PEP8 / style changes.  The goal is that reviewers see only changes related to the relevant patch.